File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.java
import java.util.Scanner;
public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        int f = in.nextInt();
        int[] array = new int[n];
        for(int i = 0; i < n; i++){
            array[i] = in.nextInt();
            if(array[i] < f)
                System.out.printf("%d ",array[i]);
        }
    }
}

File: 백준/Bronze/10871. X보다 작은 수/X보다 작은 수.kt
import java.util.Scanner
fun main(){
    val sc = Scanner(System.`in`)
    var num = sc.nextInt()
    var n = sc.nextInt()
    for(i in 0..(num-1)){
        var a = sc.nextInt()
        if(a < n)
        print("$a ")
    }
    
}